README.md

```diff
@@ -4,38 +4,76 @@ Pushing a new design and frontend concept to ownCloud
 
 ![image](https://user-images.githubusercontent.com/25989331/63966638-fd4e0080-ca9b-11e9-931a-8dd9bf3ba82f.png)
 
-[See this online Demo](https://phoenix.owncloud.com/custom/phoenix/index.html#/login)
+[See this online Demo](https://phoenix.owncloud.com/custom/phoenix/index.html#/login) (user: demo, password: demo)
 
-## How to build
+## Setting up
 
-- Run `yarn install` to build core
-- Run `yarn install-all` to install dependencies of all apps and core
-- Run `yarn dist` to build all apps configured in config.json
+Decide on which host and port Phoenix will be running, for example `https://phoenix-host:8300/phoenix-path/`.
+In this document, we will refer to the following:
+- `<phoenix-url>` as the full URL, for example `https://phoenix-host:8300/phoenix-path/`
+- `<phoenix-domain>` as the protocol, domain and port, for example: `https://phoenix-host:8300`
+
+### Setting up the ownCloud Server
+
+Make sure you have an [ownCloud Server](https://owncloud.org/download/#owncloud-server) already installed.
+
+#### Adjusting config.php
+
+Add the following entries to config/config.php:
+
+- tell ownCloud where Phoenix is located:
+```
+'phoenix.baseUrl' => '<phoenix-url>',
+```
 
-## ownCloud server prerequisites
-### When running with oauth2
-- installed and enabled oauth2 app
-- entry for Phoenix in User Authentication in admin settings with Redirection URI: `<phoenix-env>/oidc-callback.html`
-- enabled CORS domain entry for Phoenix in config.php:
+- add a CORS domain entry for Phoenix in config.php:
 ```
-'cors.allowed-domains' => [
-'<phoenix-env>',
-]
+'cors.allowed-domains' => ['<phoenix-domain>'],
 ```
-- when developing against unstable APIs (technical preview), these need to be enabled in the server core:
+
+- optional: when developing against unstable APIs (technical preview), these need to be enabled in the server core:
 ```
 dav.enable.tech_preview => true,
 ```
 
-## Create config.json
-- For reference look into config.json.sample (example with oauth2)
+#### Setting up OAuth2
+
+To connect to the ownCloud server, it is necessary to set it up with OAuth2.
+
+Install and enable the [oauth2 app](https://marketplace.owncloud.com/apps/oauth2):
+```bash
+% occ market:install oauth2
+% occ app:enable oauth2
+```
+
+Login as administrator in the ownCloud Server web interface and go to the "User Authentication" section in the admin settings and add an entry for Phoenix as follows:
 
-## Run
+- pick an arbitrary name for the client
+- set the redirection URI to `<phoenix-url>/oidc-callback.html`
+- make sure to take note of the **client identifier** value as it will be needed in the Phoenix configuration later on
+
+### Setting up Phoenix
+
+In the local Phoenix checkout, copy the `config.json.sample` file to `config.json` and adjust it accordingly:
+
+- Set the "server" key to the URL of the ownCloud server including path, make sure to include a **trailing slash**
```

Comment:
Kind of worried that this will break envs of those who are running ownCloud without subfolder (e. g. in docker) if we tell to always put there the slash.

Reply:
Argh, you're right. I'll clarify that trailing slash is only needed when using a subpath

Reply:
Adjusted